### PR TITLE
sane-backends 1.0.25

### PR DESCRIPTION
**Pull request by yurikoles**
Patch is [send to upstream](https://lists.alioth.debian.org/pipermail/sane-devel/2015-October/033972.html)

### CODE REVIEW

**Comment by DomT4** [comment]
> Patch is send to upstream

Please add a link to this in the formula as well. There's also a `revision 1` line on line 15 that needs to be cut here.

**Comment by yurikoles** [comment]
@DomT4, done.

**Comment by DomT4** [comment]
You got the wrong `revision` line I'm afraid :smile:. It was the other one in the formula that needed removing, heh.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

File: Library/Formula/sane-backends.rb [hunk 1]
```diff
@@ -45,17 +43,70 @@ def install
 end
 
 __END__
-diff --git a/include/sane/sane.h.orig b/include/sane/sane.h
-index 5320b4a..6cb7090 100644
---- a/include/sane/sane.h.orig
+https://lists.alioth.debian.org/pipermail/sane-devel/2015-October/033972.html
```
Comment (dunn) [inline Library/Formula/sane-backends.rb]: Could you move this URL to above the `patch :DATA` line and add a quick comment explaining what the patch is for? Thanks!

Comment (yurikoles) [inline Library/Formula/sane-backends.rb]: done

Comment (mikemcquaid) [inline Library/Formula/sane-backends.rb]: If you did: you didn't push it.

Comment (bfontaine) [inline Library/Formula/sane-backends.rb]: Ping?

**Comment by bfontaine** [comment]
Closing due to no response. Feel free to open another PR addressing the comments above. Thanks!

**Pull request closed without merging**